app/frontend/src/Settings/index.css

```
@@ -62,6 +62,8 @@
   transition: 0.3s all;
 }
 
+.MuiListSubheader-root,
```

These are global. I suspect they should be namespaced to `.settings`.

```suggestion
.settings .MuiListSubheader-root,
```